agent/consul/leader_metrics.go

```diff
@@ -116,7 +116,8 @@ func (m CertExpirationMonitor) Monitor(ctx context.Context) error {
 			return
 		}
 
-		if d < 24*time.Hour {
+		thirtyDays := 30 * (24 * time.Hour)
+		if d < thirtyDays {
 			logger.Warn("certificate will expire soon",
```

Comment:
Spoke with Cindy and I'm going to tweak the expiresSoon() function to return the number of days remaining along with the boolean so it can be used in the log message.

Reply:
After looking into this I remembered the days remaining was already there. It is part of the structured logging at the end, it has both "time_to_expiry" (how long the cert has left) and "expiration" (the date it will expire). So this is covered.

Reply:
@eikenb : Have you considered @cizh 's point 3 about suggesting a course of action?

For the root cert, the operator must handle the rotation themselves.

If the intermediate (signing) cert is emitting that message, something is wrong with the automated rotation and they should inspect their logs for other messages that may be related.

Reply:
Took a shot at those messages. Added them as an additional 'suggested_action' entry in the structured log seection.